platform/pjsip/templates/x86-qemu/mods.config

```diff
@@ -75,6 +75,7 @@ configuration conf {
 	@Runlevel(1) include embox.test.posix.ppty_test
 	@Runlevel(1) include embox.test.stdlib.bsearch_test
 	@Runlevel(1) include embox.test.stdlib.qsort_test
+	@Runlevel(1) include embox.test.stdlib.memcmp_test
```

Comment:
Rm it from here

Reply:
Hey @anton-bondarev  I've updated the code to comply with code style and removed global variables.Sorry for mistake, i'm new to this.What does rm mean? i haven't figured out how to run tests yet.Thank you for your feedback.